addons/grenadethrowing/XEH_postInitClient.sqf

```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+
+// Exit on HC
+if (!hasInterface) exitWith {};
+
+
+["SettingsInitialized", {
+    if (GVAR(Enabled)) then {
+        [true] call FUNC(init);
+    };
+
+    ["SettingChanged", {
```

Comment:
Technically there's no need to put this inside "SettingsInitialized". Both events can be handled by the same cose if moved to a local `_fnc_updateSetting`.